staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/validation/validation.go

```diff
@@ -85,24 +85,32 @@ func ValidateCustomResource(fldPath *field.Path, customResource interface{}, val
 					value = err.Value
 				}
 				max := int64(-1)
-				if i, ok := err.Value.(int64); ok {
+				if i, ok := err.Valid.(int64); ok {
 					max = i
 				}
-				allErrs = append(allErrs, field.TooLongMaxLength(errPath, value, int(max)))
+				allErrs = append(allErrs, field.TooLong(errPath, value, int(max)))
```

Comment:
I think we still want the TooLongMaxLength variety to avoid the message saying `bytes`